### DISCUSSION TITLE
minor-issue : uno-check version '1.11.0' python detection

### DISCUSSION THREAD

**Discussion by DomagojJugovich** [unoplatform/uno#11137] [General]
Just a heads up.
Everything else is fine but uno-check allways detects that Python is not present but it is installed.
Fix opens store to install python 3.10, bit is is intalled already.
I tried to install python 3.11 to see will the error go away, but it is the same.

**Comment by jeromelaban** [comment]
Thanks for the report! This is fixed in 1.12-dev version that you can install by specifiying the `dotnet tool` `--version` parameter. We're in the process of backporting this fix to 1.11.
